.github/workflows/test_pull_requests.yml

```
@@ -127,8 +127,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install --upgrade pip
-          pip install setuptools tox tox-gh-actions
-          python tools/minreq.py
+          pip install setuptools tox tox-gh-actions tox-min-req
         env:
           # tools/minreq.py sets all deps to their minumim stated versions
```

```suggestion
          # tools/minreq.py sets all deps to their minumim stated versions
```
This file is deleted in this PR, so probably this comment should be tweaked?

---

Yes. good point